### PR TITLE
fix(ci): use github actions

### PR DESCRIPTION
**Pull request by bodinsamuel**
We need multi platform docker image but circle makes that not easy. Good opportunity to move to GA.

**Workflows**:
- renderscript (test and lint)
- release
- release_docker

Also:
- Removed circle
- Combined both Dockerfile for simplicity, it was cumbersome in the Yaml to build both image separately. It increases a bit the information inside the Dockerfile but works well.
- Remove now unnecessary script (push is done by GA)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - completed
+
+env:
+  COMMIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: Release
+    needs: tests
```
Comment (millotp) [inline .github/workflows/release.yml:16]: Does this work with a job outside of this workflow ?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      - uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: .nvmrc
+          cache: yarn
+
+      - name: Release
+        run: yarn semantic-release
```
Comment (millotp) [inline .github/workflows/release.yml:29]:
```suggestion
        run: |
          yarn install
          yarn semantic-release
```
Or you can use composite actions for setup, something like [this](https://github.com/algolia/api-clients-automation/blob/main/.github/actions/setup/action.yml)

File: scripts/build.sh [hunk 3]
```diff
@@ -7,18 +7,10 @@ current=$(node -e "console.log(require('./package.json').version)")
...
-  --progress plain \
-  -t algolia/renderscript-pw-chromium:${current} \
-  --build-arg "VERSION=${current}" \
-  -f Dockerfile.pw \
-  .
-
 # Build renderscript
 
-docker build \
+docker buildx build \
```
Comment (millotp) [inline scripts/build.sh:12]: I'm getting an error when trying to build the image locally, do I need to update docker ?
`error: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")`

**Review by millotp — COMMENTED** [review]
Let's go GA !

File: scripts/build.sh (hunk 3 above)
Comment (bodinsamuel) [reply scripts/build.sh:12]: mmh good question, I think I had this warning too and just did `docker buildx create --use` solved it but I don't know what it does internally :D

File: .github/workflows/release.yml (hunk 2 above)
Comment (bodinsamuel) [reply .github/workflows/release.yml]: ah forgot the yarn install indeed

File: .github/workflows/release.yml (hunk 1 above)
Comment (bodinsamuel) [reply .github/workflows/release.yml]: dry-run seems to work (at least trigger)

Comment (millotp) [reply .github/workflows/release.yml]: Cool nice to know !

**Review by millotp — APPROVED** [review]

**Pull request merged**